spec/pages/post_index_page_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
+
+RSpec.describe 'Posts index page', type: :system do
+  let!(:user) do
+    User.create!(
+      name: 'Tim',
+      email: 'tim@example.com',
+      password: '123456',
+      confirmed_at: Time.now,
+      posts_counter: 0,
+      bio: 'Full-stack web developer from Germany'
+    )
+  end
+
+  let!(:posts) do
+    [
+      Post.create!(
+        author: user,
+        title: 'First title',
+        text: 'First text',
+        likes_counter: 0,
+        comments_counter: 0
+      ),
+      Post.create!(
+        author: user,
+        title: 'Second title',
```

Comment:
- [ ] Kindly make sure all of your tests are passing. these are falling tests.

![image](https://user-images.githubusercontent.com/65068771/163435831-64b88518-8243-4f22-9227-58fad7c94347.png)

- [ ] Also you need kindly provide instructions on running your test ass your tests were only running after using these commands: `bin/rails generate rspec:install`, `bundle exec rspec`.

Reply:
Thanks for your recommendations. Somehow when you run all the tests with bundle exec rspec the tests failed. Please, run all the tests individually like:

`bundle exec rspec spec/pages/login_page_spec.rb`
`bundle exec rspec spec/pages/post_idex_page_spec.rb`
`bundle exec rspec spec/pages/post_show_page_spec.rb`
`bundle exec rspec spec/pages/user_index_page_spec.rb`
`bundle exec rspec spec/pages/user_show_page_spec.rb`

and they will passed.